stock_mts_mto_rule/model/mrp_production.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
Please remove this and put licensing following this format https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py

Reply:
And you can remove utf-8 header